docs/en/userguide/01_User_manual/01_Setting_up_MFA.md

```diff
@@ -0,0 +1,51 @@
+title: Setting Up MFA
+summary: Running initial configuration of an MFA method for your account
+
+# Setting Up MFA
+
+#### 1. Prompted to setup MFA
```

Comment:
Do you think the difference from h1 to h4 is odd? Accessibility checkers would usually flag this as a problem

Reply:
This was a bit of a force of habit from the heading styles on GitHub not working so well (going from an h1 straight to an h2 there results in visually displeasing output), though I think I intended to use h3s.

Semantics are more important, though! I'll adjust this.